templates/includes/sidenav.html

```diff
@@ -1,23 +1,20 @@
 <nav class="sidenav" aria-label="Side">
     <h2>Also in this section</h2>
+    {% for page in self.sidebar_navigation %}
     <ul>
-        {# TODO one of these for each sidenav item #}
+        {% if page.active %}
         <li>
-            <a href="TODO">
-                Example page title
+            <a href="{{ BASE_URL }}{{ page.url }}" aria-current="page">
```

Comment:
It's generally better not to build URLs "string-wise" like this -- while relative URLs should be fine here, but if you do want full ones,  use [`page.get_full_url()`](https://docs.wagtail.org/en/stable/reference/pages/model_reference.html#wagtail.models.Page.get_full_url) instead